priv/perf/apps/omg_load_testing/lib/load_test/childchain.ex

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.LoadTesting.LoadTest.ChildChain do
+  @moduledoc """
+  Load test for childchain
+  """
+  use Chaperon.LoadTest
+
+  @concurrent_session 100
+  @transactions_per_session 100
+
+  def scenarios,
```

Comment:
check the `elixir-omg` credo definition. `()` are mandatory even for zero arity functions. let's try to keep coding styles consistent